mm2src/coins/nft/storage/wasm/nft_idb.rs

```diff
@@ -5,16 +5,32 @@ use mm2_db::indexed_db::{DbIdentifier, DbInstance, DbLocked, IndexedDb, IndexedD
 
 const DB_NAME: &str = "nft_cache";
 const DB_VERSION: u32 = 1;
+
+/// Represents a locked instance of the `NftCacheIDB` database.
+///
+/// This type ensures that while the database is being accessed or modified,
+/// no other operations can interfere, maintaining data integrity.
 pub type NftCacheIDBLocked<'a> = DbLocked<'a, NftCacheIDB>;
 
+/// Represents the IndexedDB instance specifically designed for caching NFT data.
+///
+/// This struct provides an abstraction over the raw IndexedDB, offering methods
+/// to interact with the database and ensuring that the database is initialized with the
+/// required tables and configurations.
 pub struct NftCacheIDB {
+    /// The underlying raw IndexedDb instance.
     inner: IndexedDb,
 }
 
 #[async_trait]
 impl DbInstance for NftCacheIDB {
+    /// Return the static name of the database.
     fn db_name() -> &'static str { DB_NAME }
 
+    /// Initialize the `NftCacheIDB` database with the provided identifier.
+    ///
+    /// This method ensures that the database is properly set up with the correct version
+    /// and has the required tables (`NftListTable`, `NftTransferHistoryTable`, and `LastScannedBlockTable`).
```

Comment:
this doc comment is not necessary ..you can add a comment to the fn signature in trait `DbInstance` instead

Reply:
Sorry missed this note, done